content/en/docs/components/pipelines/v2/installation/quickstart.md

```diff
@@ -39,7 +39,7 @@ This step demonstrates how to deploy a KFP standalone instance into an existing
 Run the following script after replacing `PIPELINE_VERSION` with the desired version of KFP (release are listed [here][releases]):
 
 ```shell
-export PIPELINE_VERSION="2.0.0-alpha.4"
+export PIPELINE_VERSION="2.0.0"
```

Comment:
Thank you for the contribution! Let's try to use a variable `{{% pipelines/latest-version %}}` instead to avoid future out-of-date.

Reference: https://github.com/kubeflow/website/blob/8dbdc966f9ea8504a74dd4a2df740b060ed8d72b/content/en/docs/components/pipelines/v1/installation/standalone-deployment.md?plain=1#L76

Reply:
It looks good to me.

@zijianjoy do you approve this to be merged?